kernel/vmemitcodes.ml

```diff
@@ -135,6 +135,10 @@ let out env opcode =
 
 let is_immed i = Uint63.le (Uint63.of_int i) Uint63.maxuint31
 
+let is_accu_dead = function
```

Comment:
Maybe a comment here to restate what the commit message says.

Reply:
Do you want to add it, or should I merge this PR as is?

Reply:
I will add it.